Dockerfile.centos7

```diff
@@ -0,0 +1,177 @@
+# libmpc-devel, and file (see above)
+
+# NOTE: Q: What about devtoolset-7? Will that work instead?
+#       A: Not quite:
+#          https://stackoverflow.com/questions/49393888/how-can-i-use-the-new-c-11-abi-with-devtoolset-7-on-centos-rhel
+#          (tl;dr: devtoolset-7 does not support the new cxx11 ABI since it
+#          conflicts with CentOS sys libs.)
+#          Rapids will use new new ABI for its binaries, including its own
+#          libstdc++, and the rest of CentOS will continue to use the
+#          system default libs.
+
+ARG GCC7_DIR=${RAPIDS_SRC_DIR}/gcc7
+
+RUN mkdir -p ${GCC7_DIR}
+RUN cd ${GCC7_DIR} && wget -q http://ftp.gnu.org/gnu/gcc/gcc-7.2.0/gcc-7.2.0.tar.gz 
+RUN cd ${GCC7_DIR} && tar zxf gcc-7.2.0.tar.gz
+RUN cd ${GCC7_DIR}/gcc-7.2.0 && \
+    ./configure --prefix=${GCC7_DIR} --disable-multilib && \
+    make -j && make install
+
+# Remove gcc source dir and tarfile
+RUN rm -r ${GCC7_DIR}/gcc-7.2.0 ${GCC7_DIR}/gcc-7.2.0.tar.gz
+
+# Update environment to use new gcc7
+ENV CC=${GCC7_DIR}/bin/gcc
+ENV CXX=${GCC7_DIR}/bin/g++
+ENV PATH=${GCC7_DIR}/bin:$PATH
+ENV LD_LIBRARY_PATH=${GCC7_DIR}/lib64:$LD_LIBRARY_PATH
+ENV CUDAHOSTCXX=${GCC7_DIR}/bin/g++
+# NOTE: Many/all of the package versions used below are defined in the
+# "args" insertfile
+
+# Install conda
+## Build combined libgdf/pygdf conda env
+RUN curl ${MINICONDA_URL} -o /miniconda.sh && \
+    sh /miniconda.sh -b -p /conda && \
+    rm -f /miniconda.sh && \
+    conda update -y -n base -c conda-forge conda
+
+# Add a condarc to remove blacklist
+ADD .condarc-cuda${CUDA_SHORT_VERSION} /conda/.condarc
+
+RUN conda create --no-default-packages -n gdf \
+      python=${PYTHON_VERSION} \
+      anaconda-client \
+      arrow-cpp=${ARROW_CPP_VERSION} \
+      cmake=${CMAKE_VERSION} \
+      cmake_setuptools \
+      conda-build \
+      conda-verify \
+      cffi=${CFFI_VERSION} \
+      cmake=${CMAKE_VERSION} \
+      cython=${CYTHON_VERSION} \
+      flake8 \
+      make \
+      numba=${NUMBA_VERSION} \
+      numpy=${NUMPY_VERSION} \
+      pandas=${PANDAS_VERSION} \
+      pyarrow=${PYARROW_VERSION} \
+      pytest \
+      pytest-cov \
+      scikit-learn=${SKLEARN_VERSION} \
+      scipy=${SCIPY_VERSION} \
+      conda-forge::blas=1.1=openblas \
+      libgcc-ng=${LIBGCC_NG_VERSION} \
+      libgfortran-ng=${LIBGFORTRAIN_NG_VERSION} \
+      libstdcxx-ng=${LIBSTDCXX_NG_VERSION} \
+    && conda clean -a && \
+    chmod 777 -R /conda
+
+# Special case: libcumlmg is not available for CUDA 9.2
+RUN if [ "${CUDA_MAJORMINOR_VERSION}" != "9.2" ]; then conda install -n gdf -y --no-deps -c ${NVIDIA_CONDA_LABEL} -c conda-forge libcumlmg; fi
+
+# Enables "source activate conda"
+SHELL ["/bin/bash", "-c"]
+
+
+# Change LD_LIBRARY_PATH to exclude /usr/local/cuda/* since
+# numba.cuda cannot load that, and instead have it load
+# /lib64/libcuda.so instead
+ENV LD_LIBRARY_PATH=${GCC7_DIR}/lib64:$LD_LIBRARY_PATH_ORIG:$CONDA_PREFIX
```

Comment:
Line 119 also modified this with the same `GCC7_DIR`

Reply:
Thanks, I changed this and (hopefully) made other changes to make it clearer as to what's going on.